README.md

```diff
@@ -303,7 +303,7 @@
 
 ## Module description
 
-[Apache HTTP Server][] (also called Apache HTTPD, or simply Apache) is a widely used web server. This [Puppet module][] simplifies the task of creating configurations to manage Apache servers in your infrastructure. It can configure and manage a range of virtual host setups and provides a streamlined way to install and configure [Apache modules][].
+[Apache HTTP Server][] (also called Apache HTTPD, or simply Apache) is a widely used web server. This [Puppet module][] simplifies the task of creating configurations to manage Apache servers in your infrastructure. It can configure and manage a range of virtual host setups and provides a streamlined way to install and configure [Apache modules][]. Please note that this module has went through an update and is now supporting only Apache 2.4 and above. For Apache < 2.4 please use an older version of the module.
```

Comment:
Should the last two sentences marked as bolt text? That helped a bit for Vox Pupuli releases. People didn't see the announcement of the breaking change within one paragraph.